### DISCUSSION TITLE
Basic Tree View example does not work with StackBlitz Clarity Light Theme V5

### DISCUSSION THREAD

**Discussion by dstovall** [vmware/clarity#6659] [Q&A]
I've got a more complicated question (or maybe it's a bug report) that I'm trying to implement in StackBlitz, but I'm having trouble even getting started.

I started with a clone of [Clarity Light Theme V5](https://stackblitz.com/edit/clarity-light-theme-v5) and simply dropped in the very first code sample from the [Tree View documentation](https://clarity.design/angular-components/tree-view/).

This results in a tree, but the tree is fully expanded and doesn't respond to clicking the arrows/handles.
https://stackblitz.com/edit/clarity-tree-view-basic-tree

I've found at least one other example that works, but it's using Clarity 1.x.  At least it proves (to me) that trees can work in my browser with StackBlitz.

Any suggestions on getting this example moving?

Thanks,
Drew

**Comment by dstovall** [comment]
When I attempted the same process with the [Clarity Light Theme V4](https://stackblitz.com/edit/clarity-v4-light-theme), the application works as expected...

https://stackblitz.com/edit/clarity-v4-tree-view-basic-tree

How do I determine if this is an issue with the Angular component itself or with the StackBlitz template?

**Comment by Jinnie** [comment]
Hi @dstovall 
There's an error in the browser console that can help diagnosing the problem.
Please add the `BrowserAnimationsModule` import to your `app.module.ts`. All should be fine after this.
